src/test/csi-neonsan/templates/controller-rbac.yaml

```diff
@@ -0,0 +1,253 @@
+apiVersion: v1
```

Comment:
Copyright annotation

Reply:
The other charts  has no Coypright annotaion, so i delete

Reply:
In helm/chart https://github.com/helm/charts/blob/master/stable/nfs-server-provisioner/templates/rolebinding.yaml, it is not write down copyright part. So we need not add it.